docs/glossary.md

```diff
@@ -32,7 +32,7 @@ Acronyms and Abbreviations
  LRN                 Local Response Normalization                     
  mAP                 Mean Average Precision                           
  Intel® OneDNN       Intel® OneAPI Deep Neural Network Library        
- MO                  Model Optimizer                                  
+ `mo`                  Command-line tool for model conversion, CLI for `tools.mo.convert_model`                             
  MVN                 Mean Variance Normalization                      
```

Comment:
May be it make sense to add `convert_model` and `Model Conversion API`?

Reply:
Have added the information with the [cd8a57d](https://github.com/openvinotoolkit/openvino/pull/17454/commits/cd8a57d71c71f69719c3254eb49f323d5c23a635) commit.